### PR TITLE
create modal module and finish display data function, bind add book, validate form 

### LINKED ISSUE #3: Build book view
Build view of app to user interact with

### PR DESCRIPTION
**Pull request by 2H1EU8**

### CODE REVIEW

File: src/views/modules/BookModal.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        type="text"
+        name="book-category"
+        id="book-category"
+        value="${category || ''}"
+        >
+
+        <div class="status-wrap">
+          <label for="active">Active</label>
+          <input
+          ${status ? 'checked' : ''}
```
Comment (thanhbui27) [inline src/views/modules/BookModal.js:47]: U can use && instead ?

Comment (thanhbui27) [inline src/views/modules/BookModal.js:47]: U can use && instead ?

File: src/views/modules/BookModal.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+function BookModal({
+  id = null,
+  title = null,
+  author = null,
+  category = null,
+  status = false,
+  number = null,
+  price = null,
+}) {
```
Comment (thanhbui27) [inline src/views/modules/BookModal.js:9]: I think you should use a specific default value here

**Comment by thanhbui27** [comment]
pls click resolve conversation if u fix them

File: src/helper/formValidate.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      e.target.setCustomValidity('Author cannot be empty!');
+    }
+  });
+
+  input.addEventListener('change', (e) => {
+    e.target.setCustomValidity('');
+  });
+};
+
+export const numberValidate = (form, name) => {
```
Comment (thanhbui27) [inline src/helper/formValidate.js]: These three functions are quite similar. can you combine 3 functions into 1?

Comment (2H1EU8) [reply src/helper/formValidate.js]: ok I'll give it a try.

**Pull request merged**